### DISCUSSION TITLE
Visual Studio/Code

### DISCUSSION THREAD

**Discussion by spinunet** [Arduino-IRremote/Arduino-IRremote#845] [Q&A] [answered]
I'd like to build this in Visual Studio 2017/2019 or VS Code.  Is this possible?

**Comment by AnalysIR** [comment, marked as answer]
I believe it may be possible & at least one of the solution is a paid product. I did a quick search online and some solutions came up.

However, given that you had to ask this question, I suggest you consider sticking with the standard IDEs from Arduino(several) or platform.io

**Reply by spinunet** [reply]
Thanks AnalysIR. I appreciate your time and reply.
